google/cloud/security/common/gcp_api/compute.py

```diff
@@ -656,6 +678,28 @@ def get_forwarding_rules(self, project_id, region=None):
                                                        'forwardingRules')
         return results
 
+    def get_global_operations(self, project_id, operation_id):
+        """Get the Operations Status
+        Args:
+            project_id (str): The project id.
+            operation_id (str): The operation id.
+
+        Returns:
+            dict: Global Operation status and info.
+            https://cloud.google.com/compute/docs/reference/latest/globalOperations/get
+
+        Raises:
+            apiError: Returns if the api is not enabled or executable.
```

Comment:
Nit: Should list ApiNotEnabledError and ApiExecutionError separately in the Raises section.